packaging/supervisor.conf

```diff
@@ -89,5 +89,16 @@ user=dd-agent
 autorestart=unexpected
 exitcodes=0
 
+[program:logs-agent]
+command=/opt/datadog-agent/bin/logs-agent --ddconfig /etc/dd-agent/conf.d
```

Comment:
Although totally unrelated to this PR, it would be nice to have a common interface for our binary launching arguments. The switch we use on agent6 for the config locations I believe is `--confd`.

Reply:
Good point, it could definitely be updated. However this is a temporary parameter. The plan is to merge `logs-agent` into `datadog-agent` as soon as agent 6 is out, and when we do this we won't need this parameter anymore.